src/data/markdown/translated-guides/en/07 Misc/05 k6 Extensions.md

```diff
@@ -1,4 +1,197 @@
 ---
 title: 'k6 Extensions'
-redirect: 'https://k6.io/blog/extending-k6-with-xk6'
----
\ No newline at end of file
+---
+
+Traditionally, extending k6 with custom functionality that isn't available in the
```

Comment:
In Docs, I'd remove the whole historical intro. 

Additionally, I would add a label about version support like in Scenarios.

Reply:
Hhmm I think explaining the issues extensions were meant to solve serves as a good introduction for why someone might want to use them. Starting from "What are k6 extensions?" would be too abrupt IMO. Unless you have another suggestion for the intro :)

And sure, I'll add a more prominent version support notice.

Reply:
https://github.com/grafana/k6-docs/pull/346/files#r678100601